src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-units/attachment-units.component.ts

```diff
@@ -0,0 +1,200 @@
+import { Component, OnInit } from '@angular/core';
+import { faBan, faClock, faExclamationTriangle, faGlobe, faPlus, faQuestionCircle, faSave, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { ActivatedRoute, Router } from '@angular/router';
+import { HttpErrorResponse } from '@angular/common/http';
+import { onError } from 'app/shared/util/global.utils';
+import { AttachmentUnitService } from 'app/lecture/lecture-unit/lecture-unit-management/attachmentUnit.service';
+import { combineLatest } from 'rxjs';
+import dayjs from 'dayjs/esm';
+import { objectToJsonBlob } from 'app/utils/blob-util';
+import { AlertService } from 'app/core/util/alert.service';
+import { TranslateService } from '@ngx-translate/core';
+
+type LectureUnitDTOS = {
+    unitName: string;
+    releaseDate?: dayjs.Dayjs;
+    startPage: number;
+    endPage: number;
+};
+
+type LectureUnitInformationDTO = {
+    units: LectureUnitDTOS[];
+    numberOfPages: number;
+    removeBreakSlides: boolean;
+};
+
+@Component({
+    selector: 'jhi-attachment-units',
+    templateUrl: './attachment-units.component.html',
+    styleUrls: [],
+})
+export class AttachmentUnitsComponent implements OnInit {
```

Comment:
Optional: I'd suggest naming this component a little more clearly: `AttachmentUnitProcessingComponent`? Do the other reviewers agree?